apps/web/src/app/globals.css

```
@@ -129,11 +129,13 @@
 
   /* ===== Typography ===== */
 
-  /* Font Families - Note: Actual fonts need to be loaded via Typekit */
-  --font-family-title: quasimoda, acumin-pro, Montserrat, Verdana, sans-serif;
-  --font-family-body: montserrat, -apple-system, system-ui, BlinkMacSystemFont, "Segoe UI", Roboto, "Helvetica Neue", Arial, sans-serif;
+  /* Font Families
+     - Montserrat & IBM Plex Mono: self-hosted via next/font/google (CSS vars injected on <html>)
+     - Quasimoda & Stenciletta: loaded via Adobe Typekit (afterInteractive) */
+  --font-family-title: quasimoda, -apple-system, system-ui, BlinkMacSystemFont, "Segoe UI", Roboto, "Helvetica Neue", Arial, sans-serif;
```

Stylelint (17.6.0)

[error] Expected "BlinkMacSystemFont" to be "blinkmacsystemfont" (value-keyword-case)

(value-keyword-case)